README.md

```diff
@@ -0,0 +1,113 @@
+## Requirements
+- Node.js v12.22.4
+- npm/npx v7.21.1
+- Docker v20.10.8
+- Cairo 0.5.1
+
+## Install Dependencies
+```
+npm install
+```
+
+## Use
+Make sure docker is running locally
+Generate the artifacts by compiling the contract using staknet-compile:
```

Comment:
Check how this file looks when it's displayed, lines 13 and 14 are displayed in the same line.